dataRetriever/mock/nodesCoordinatorStub.go

```diff
@@ -1,8 +1,11 @@
 package mock
 
+import "github.com/ElrondNetwork/elrond-go/sharding/nodesCoordinator"
+
 // NodesCoordinatorStub -
 type NodesCoordinatorStub struct {
```

Comment:
just a reminder, to evaluate using sharding mocks from testscommon here, maybe when the feat branch is ready